src/services/utilServices/RateLimiter.ts

```diff
@@ -13,8 +16,21 @@ const DEFAULT_AUTH_TOKEN_EXPIRY_MILLISECONDS = 900000
 // but not on the other, leading to inconsistent behaviour.
 // eslint-disable-next-line import/prefer-default-export
 export const rateLimiter = rateLimit({
-  windowMs: config.get("auth.tokenExpiry") / (1000 * 60),
-  max: 100, // Limit each IP to 100 requests per `window` (here, per 15 minutes)
+  windowMs: config.get("auth.tokenExpiryInMs"),
```

Comment:
i think you might have made a mistake here - going off the prod env var, this is `86400000 ms`, which is a day.

i think this token expiry is different from the rate limits - we want to limit incoming auth requests to a max of 100 requests per window, this has nothing to do with the token expiry, which is how long a user can stay signed in for before we invalidate their session.

could we instead just use `DEFAULT_AUTH_TOKEN_EXPIRY_MILLISECONDS` defined above? (an hour or 15 minutes is both fine. i think 15 might be more permissive but this isn't likely to be ddos or abused given the low rate)

Reply:
sure modified to 15 mins [here](https://github.com/isomerpages/isomercms-backend/pull/1183/commits/168074ec0971e3083649f157e7bf6d06f5c81a31)